cmd/manager/main.go

```diff
@@ -149,8 +155,11 @@ func main() {
 	log.Info("Starting the Cmd.")
 
 	// Start all registered controllers
-	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
-		log.Error(err, "Manager exited non-zero")
-		os.Exit(1)
+	return mgr.Start(stopCh)
+}
+
+func main() {
+	if err := start(); err != nil {
+		panic(err)
 	}
```

Comment:
@mjudeikis this section might be causing some problems in the staging environment. Here's the output from the pod:

```
[root@hive-stage-master-ip-10-140-2-38.ec2.internal ~]# oc get pods -n certman-operator
NAME                                READY     STATUS    RESTARTS   AGE
certman-operator-7d755788c8-5g7ns   0/1       Error     2          31s
certman-operator-catalog-5ln5b      1/1       Running   0          13m

[root@hive-stage-master-ip-10-140-2-38.ec2.internal ~]# oc logs certman-operator-7d755788c8-5g7ns  -n certman-operator
{"level":"info","ts":1575422149.6017106,"logger":"cmd","msg":"Go Version: go1.11.13"}
{"level":"info","ts":1575422149.6018276,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1575422149.6018362,"logger":"cmd","msg":"Version of operator-sdk: v0.8.2"}
{"level":"info","ts":1575422149.6021032,"logger":"leader","msg":"Trying to become the leader."}
{"level":"error","ts":1575422149.6386,"logger":"cmd","msg":"","error":"Unauthorized","stacktrace":"github.com/openshift/certman-operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\tsrc/github.com/openshift/certman-operator/vendor/github.com/go-logr/zapr/zapr.go:128\nmain.start\n\tsrc/github.com/openshift/certman-operator/cmd/manager/main.go:87\nmain.main\n\tsrc/github.com/openshift/certman-operator/cmd/manager/main.go:162\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:201"}
panic: Unauthorized
goroutine 1 [running]:
main.main()
        src/github.com/openshift/certman-operator/cmd/manager/main.go:163 +0x44
```

Could you help with debugging this?

Reply:
FYI: Merge was reverted to bring staging back online.

Reply:
Can I get access to staging to deploy manual build somehow? It a challenge to debug it it like this.

Reply:
Error looks like leader election failed. Did you had old version still running?

Reply:
I tried deploying your changes in another stage cluster, and it works. So there probably isn't an issue with your code after all. There must be an issue with how it's deployed to hive-staging using OLM. I'm following up with my team about debugging OLM.